discovery/consul/consul_test.go

```diff
@@ -68,6 +68,50 @@ func TestConfiguredServiceWithTag(t *testing.T) {
 	}
 }
 
+func TestConfiguredServiceWithTags(t *testing.T) {
```

Comment:
Table-driven tests would be more readable here but this is my personal taste.

Reply:
:+1: agreed, done